src/wmtk/SimplicialComplex.hpp

```diff
@@ -13,15 +13,15 @@ namespace internal {
 
 struct SimplexLessFunctor
 {
-    const Mesh& m;
+    const Mesh* m;
```

Comment:
What was the impetus for this?

Reply:
SimplicialComplex was not assignable due to the const reference.